rfcs/text/0045/vulnerbility.yml

```diff
@@ -0,0 +1,104 @@
+- name: vulnerability
+  title: Vulnerability
+  group: 2
+  short: These fields contain information about vulnerabilities.
+  description: >
+    These fields contain information about vulnerabilities identified in assets.
+    They help in tracking, mitigating, and documenting vulnerabilities.
+
+  
+  fields:
+    - name: vendor.id
+      level: extended
+      type: keyword
+      description: >
+        A vulnerability doesn't necessarily have a CVE associated with it. It makes sense to separate vulnerability ID (like CVEs) to the vendor/detection IDs.
+
+    - name: title
+      level: extended
+      type: keyword
+      description: >
+        Title/Name/Short Description for vulnerability, to be used in flyout and dashboards.
+
+    - name: mitigation
+      level: extended
+      type: text
+      description: >
+        Explains how to fix or mitigate the problem, could be used to store resolution from the scanner vendor or document mitigation in place.
+
+    - name: published
+      level: extended
+      type: date
+      description: >
+        The “published” field indicates the date when information about a specific vulnerability was publicly disclosed or made available. It represents the moment when details about the vulnerability were shared with the security community, vendors, and the public. This field helps security professionals track the timeline of vulnerability awareness, in ISO 8601 format - YYYY-MM-DD.
+
+    - name: patch
+      type: group
+      description: >
+        Information about patches available for the vulnerability.
+      fields:
+        - name: exists
+          level: extended
+          type: boolean
+          description: >
+            Indicates whether a security fix or update (commonly known as a patch) is available to address the identified vulnerability.
+
+        - name: name
+          level: extended
+          type: text
+          description: >
+            Name of the patch.
```

Comment:
Can you give an example of a named patch? I'm not sure follow what would go into this field.